codebuild/common-posix.sh

```diff
@@ -31,8 +31,8 @@ if [ "$TRAVIS_OS_NAME" != "osx" ]; then
     sudo apt-get install libssl-dev -y
     install_library s2n 7c9069618e68214802ac7fbf45705d5f8b53135f
 fi
-install_library aws-c-common
-install_library aws-c-io
+install_library aws-c-common v0.3.7
```

Comment:
Was this intended to be temporary?

Reply:
no, I want to pin these long term, we should not have a downstream build that isn't pinned, it leads to CI dying due to no fault of the comitter.